test/controllers/util/DynamicFormUtilsSpec.scala

```diff
@@ -171,28 +172,54 @@ class DynamicFormUtilsSpec extends AnyFlatSpec with MockitoSugar with BeforeAndA
     dateField.fieldErrors should equal(List(s"There was no number entered for the Day."))
   }
 
-    "validateAndConvertSubmittedValuesToFormFields" should "return a 'whole number'-related error for the numeric fields that are missing values" in {
-      //Shouldn't be possible on client-side due to "type:numeric" attribute added to HTML, but in case that is removed
-      val (_, dynamicFormUtils): (ListMap[String, List[String]], DynamicFormUtils) = generateFormAndSendRequest(
-        MockFormValues(month = List("b4"),
-          year = List("2r"),
-          numericTextBoxValue = List("1.5"),
-          day2 = List("000"),
-          month2 = List("e"),
-          year2 = List("-")
-        )
+  "validateAndConvertSubmittedValuesToFormFields" should "return an error when future date is not allowed" in {
+    val dateTime = LocalDateTime.now().plusDays(1)
```

Comment:
Actually, we are already testing all these scenarios in the `FormFieldSpec.scala` class. So, I just kept only one test to avoid writing too many duplicate tests. Hope that makes sense.